### PR TITLE
fix: text color on footer

### PR DESCRIPTION
**Pull request by himanshu1221**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

There was a problem with the footer text color  it was too light to be visible as discussed I changed the color to black and now it is much more visible

### Test plan (required)

I have tested it in a test server and its working perfectly fine
Screenshot from 2023-04-06 12-51-06

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

**Comment by himanshu1221** [comment]
> Hey @himanshu1221 , It looks better now. Thanks! Can social media icons stay as they were before? They already have a hover state.

sure I'll edit it and make a commit

**Comment by himanshu1221** [comment]
> Hey @himanshu1221 , It looks better now. Thanks! Can social media icons stay as they were before? They already have a hover state.

I have made the suggested hanged you may have  a look

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (omeraplak) [inline .gitpod.yml:1]: Gitpod configuration can be deleted.

**Review by omeraplak — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (himanshu1221) [reply .gitpod.yml]: Done

**Review by omeraplak — APPROVED** [review]

**Pull request merged**